WebSocketTest/Decoders/JsonDecoder.cs

```diff
@@ -1,17 +1,15 @@
 ﻿using WebSocketTest.Datatypes;
 using System.Text.Json;
 
-
-
 namespace WebSocketTest.Decoders
 {
-	static class JsonDecoder
-	{
-		static public Inputs ParseToInputs(Inputs target)
-		{
-                string json = JsonSerializer.Serialize(target);
-                // Converts and returns json string to Inputs object
-                return JsonSerializer.Deserialize<Inputs>(json); 
-		}
-	}
-}
+    static class JsonDecoder
```

Comment:
You changed everything to spaces instead of tabs...
![afbeelding](https://user-images.githubusercontent.com/31539215/66302757-cc15eb00-e8f9-11e9-8ec7-50a053ac3522.png)